proposals/lip-0003.md

```diff
@@ -75,35 +78,32 @@ Given the study of the issue presented in the previous section, this LIP propose
+- The delegate's unique identifier is the binary address of the delegate. Note that this proposal does not depend on the implementation of [LIP-0018](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0018.md). The only requirement here is that the addresses have to be unique for each forging delegate which is ensured by the protocol in any case.
+- The round's unique identifier is one of the two random seed generated by the specifications in [LIP-0022](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0022.md#random-seeds-computation).
 
-```js
-modules.rounds.calc(height).toString();
-```
 
-By having a unique hash per delegate for every round, we will ensure that in every round the list of delegates will be re-ordered based on a uniform distribution.
+By having a unique hash per delegate for every round, we will ensure that in every round the list of delegates will be re-ordered based on a uniform distribution. Moreover, these changes not only fix the uniformity issue (caused by the two issues raised in the [Motivation](#motivation) section), but it will also allow simpler and more human-readable code in any implementation.
 
 ## Specification
 
-The changes will only affect the `Delegates.prototype.generateDelegateList` function mentioned above. These changes consist of removing the nested `for` loops and generating a new list of “delegate hashes” to be re-calculated and ordered every round. In particular, the proposed fix implements a new algorithm that goes as follows:
+The changes affect the `shuffleDelegateListForRound` function mentioned above. These changes consist of removing the nested `for` loops and generating a new list of “delegate hashes” to be re-calculated and ordered every round. In particular, the proposed fix is as follows:
 
-1. The variable `seedSource` is generated in the same way as now:
+1. The variable `seedSource` is generated as follows:
 
 	```js
-	var seedSource = modules.rounds.calc(height).toString();
+	const seedSource = randomSeed1;
 	```
+	where `randomSeed1` is the variable specified in [LIP-0022](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0022.md#random-seeds-computation). 
 
-2. One hash is generated per delegate using `seedSource` and the delegate public key (or ID). For simplicity, we propose to use the hashing function already utilised in the existing code, although the final choice here is left to the reference implementation. Example given in pseudo-code:
+2. One hash is generated per delegate using `seedSource` and the delegate address. For simplicity, we propose to use SHA-256, which is the hashing function already utilised in the existing function. Following the current implementation, an example in pseudo-code will be:
 
 	```
-	forEach delegate
-		delegate.pubKeyHash = createHash(seedSource + delegate.publicKey)
+	forEach delegate in delegateList
+		delegate.AddressHash = hash(seedSource + delegate.address)
```

Comment:
Maybe `delegate.roundHash` would be a more suitable name, as the hash does not only depend on the address. But just a suggestion.

Reply:
also I would not capitalize the variable name `AddressHash --> addressHash`

Reply:
changed to Jan's suggestion